docs/tutorial-model-and-deploy.md

```diff
@@ -185,8 +224,11 @@ Provide values for the properties defined with `get_input: …` (see [4. Set the
 
 | Property | Value | Explanation |
 |:---------|:------|:------------|
-| DockerEngineURL | `tcp://dind:2375` or `tcp://${PUBLIC_HOSTNAME}:2222` | The address of the dind container. |
-| Port     | `9990` | A port between `9990` and `9999` as only these ports are exposed by default in the dind container. (May be changed in the docker-compose file.) |
+| DockerEngineURL | `tcp://${PUBLIC_HOSTNAME}:2222` | The address of the (host of the) dind container. |
```

Comment:
Maybe add an example here in the explanation like `tcp://192.168.1.5:2222`

Reply:
I have added an example in step 1 and specified again, which part of the string needs to be replaced.